lib/SIL/SILFunctionType.cpp

```diff
@@ -148,22 +150,68 @@ CanSILFunctionType SILFunctionType::getWithoutDifferentiability() {
                               getOptionalErrorResult(), getASTContext());
 }
 
+// Returns the canonical generic signature for an autodiff associated function
+// given an existing associated function generic signature. All differentiation
+// parameters are constrained to conform to `Differentiable`.
+static CanGenericSignature getAutoDiffAssociatedFunctionGenericSignature(
+    CanGenericSignature assocFnGenSig,
+    ArrayRef<SILParameterInfo> originalParameters,
+    AutoDiffIndexSubset *parameterIndices, SILModule &module) {
+  // If associated function has no
```

Comment:
Incomplete comment.

Reply:
Thanks for the catch! I'll update this in my next PR.

Reply:
Removed in https://github.com/apple/swift/pull/26448.